addon/commands/insert-html-command.ts

```diff
@@ -24,7 +24,7 @@ export default class InsertHtmlCommand extends Command {
     const parser = new DOMParser();
     const html = parser.parseFromString(htmlString, 'text/html');
     const bodyContent = html.body.childNodes;
-    const reader = new HtmlReader(this.model);
+    const reader = new HtmlReader(this.model, true);
```

Comment:
think I should create a `WhitespaceCollapsingHtmlReader that extends `HtmlReader` instead